src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -876,7 +876,18 @@ Regards the latter, run `mvn apache-rat:check` to verify all files are suitably
 See link:http://search-hadoop.com/m/DHED4dhFaU[HBase, mail # dev - On recent discussion clarifying ASF release policy]
 for how we arrived at this process.
 
-To help with the release verfication `hbase-vote.sh` shell script is available under `dev-support` directory of HBase source. Following are the usage details.
+To help with the release verification, please follow the guideline below and vote based on the your verification.
+
+=== Baseline Verifications for Voting Release Candidates
+
+Although contributors have their own checklist for verifications, the following items are usually used for voting on release candidates.
+
+* CHANGES.md if any
+* RELEASENOTES.md (release notes) if any
+* Generated API compatibility report
+** For what should be compatible please refer the [versioning guideline](https://hbase.apache.org/book.html#hbase.versioning), especially for items with marked as high severity
```

Comment:
this didn't render correctly as a link in the qabot build of the ref guide.

Reply:
fixed, thanks for calling out